<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>834</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="interface_frame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>350</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame { background-color: white; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QProgressBar" name="interface_battery">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>350</y>
       <width>20</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QFrame" name="interface_screen">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>191</width>
       <height>151</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QListWidget" name="interface_menu_selection">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget { background-color: transparent; }</string>
      </property>
      <item>
       <property name="text">
        <string>NEW SESSION</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SESSION LOG</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TIME AND DATE</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </item>
     </widget>
     <widget class="QTimeEdit" name="interface_menu_selection0_time">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTimeEdit {
                border: none;
                background: transparent;
            }
QTimeEdit::up-button, QTimeEdit::down-button {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 0px;
}
QTimeEdit::up-arrow, QTimeEdit::down-arrow {
    image: none;
}</string>
      </property>
      <property name="displayFormat">
       <string>m:ss</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="interface_menu_selection0_progress">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>150</width>
        <height>15</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QListWidget" name="interface_menu_selection1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>170</width>
        <height>130</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QFrame" name="interface_menu_selection0_completed">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>100</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Completed!</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="completed_exit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>40</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>EXIT</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="interface_menu_selection2_frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>170</width>
        <height>130</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame { background-color: white; }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QComboBox" name="interface_menu_selection2_combo">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Day</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hour</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Minute</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Second</string>
        </property>
       </item>
      </widget>
      <widget class="QTimeEdit" name="interface_menu_selection2_time">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTimeEdit {
                border: none;
                background: transparent;
            }
QTimeEdit::up-button, QTimeEdit::down-button {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 0px;
}
QTimeEdit::up-arrow, QTimeEdit::down-arrow {
    image: none;
}</string>
       </property>
       <property name="displayFormat">
        <string>hh:mm:ss</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="interface_menu_selection2_date">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit {
                border: none;
                background: transparent;
            }
QDateEdit::up-button, QDateEdit::down-button {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 0px;
}
QDateEdit::up-arrow, QDatedit::down-arrow {
    image: none;
}</string>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
      <widget class="QPushButton" name="interface_menu_selection2_increase">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>40</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="interface_menu_selection2_decrease">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>40</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="interface_menu">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Menu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="interface_power">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ON/OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="interface_pause">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PAUSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="interface_start">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>270</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QPushButton" name="interface_stop">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="interface_blue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>15</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGraphicsView{background-color:grey;}</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="interface_green">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>15</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGraphicsView{background-color:grey;}</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="interface_red">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>15</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGraphicsView{background-color:grey;}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="interface_recharge">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>360</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ReCharge</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="eeg_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame { background-color: white; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>EEG Connection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleConnection">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="pc_transation">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame { background-color: white; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PC Transation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pc_submit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="pc_screen">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame { background-color: white; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PC Moniotor</string>
     </property>
    </widget>
    <widget class="QLabel" name="after_baseline_freq">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>631</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="pc_monitor_show">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>231</width>
       <height>130</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="waveform_box" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>480</y>
      <width>441</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EEG Waveform Graph</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>510</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>530</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>550</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>570</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>590</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>610</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>490</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1289</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
